File: pset1/greedy.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{

float dollars=0;
int cents=0;
int coins=0;

    do{
        printf("How much do I owe you? ");
        dollars = get_float();
    }while (dollars < 0);
    cents=roundf(dollars*100);
    //printf("%i\n", cents);

while (cents - 25>=0)
{    cents=cents - 25;
     coins=coins+1;
}

while (cents - 10>=0)
{    cents=cents - 10;
     coins=coins+1;
}

while (cents - 5>=0)
{    cents=cents - 5;
     coins=coins+1;
}

if (cents >0)
{
    coins=coins+cents;
}

    printf("%i\n",coins);
}

File: pset6/greedy.py
while True:
    #print("Tell me how many rows you want")
    dollars = float(input("How much do I owe you? "))
    if dollars > 0:
        break

cents = int(dollars * 100)
coins = 0;

coins += cents // 25
cents = cents % 25

coins += cents // 10
cents = cents % 10

coins += cents // 5
cents = cents % 5

coins += cents

print(coins)